components/ui/AppCta.vue

```diff
@@ -22,34 +22,34 @@
 </template>
 
 <script setup lang="ts">
-import UiBasicLink from "~/components/ui/UiBasicLink.vue";
+import type { GeneralLink } from "~~/constants/appLinks";
 
 interface Props {
   isWider?: boolean;
   kind?: "primary" | "secondary" | "ghost";
-  label?: string;
+  label: string;
   theme?: "light" | "dark";
+  url: string | GeneralLink;
 }
 
 const props = withDefaults(defineProps<Props>(), {
   isWider: false,
   kind: "primary",
-  label: "",
   theme: "light",
 });
 
 const iconPerLinkType = computed(() => {
-  const url = this.$attrs.url;
+  const url = props.url;
 
   if (props.label === "Under construction") {
     return "error-outline-16";
   }
-  if (UiBasicLink.isExternal(url)) {
-    return "launch-16";
-  }
-  if (UiBasicLink.isIdAnchor(url)) {
-    return "arrow-down-16";
-  }
+  // if (UiBasicLink.isExternal(url)) {
```

Comment:
Why did you comment these lines?

Reply:
Sorry, I forgot to revert it. There are some issues with those functions, so they cause Typescript errors.

For now, I've added a quick fix and TODO comments with https://github.com/Qiskit/qiskit.org/pull/2980/commits/19732b456e050503badc5c204b0842b7789da37b.